db/migrations/bindata.go

```diff
@@ -4,10 +4,14 @@
 // mysql/000001_create_teams.up.sql
 // mysql/000002_create_team_members.down.sql
 // mysql/000002_create_team_members.up.sql
+// mysql/000005_create_channel_member_history.down.sql
```

Comment:
I would expect this to be `000003` right?

Reply:
I have already PRs for 000003 and 000004, sent this first because of our discussion

Reply:
Oh awesome!